packages/app/src/components/FluentChatListItem.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="absolute text-xs text-gray-500 right-0 top-0 mr-4 mt-3">
+    2 hours ago
+  </div>
+  <div
+    class="flex items-center justify-center h-10 w-10 rounded-full bg-pink-500 text-pink-300 font-bold flex-shrink-0"
+  >
+    T
+  </div>
+  <div class="flex flex-col flex-grow ml-3">
+    <div class="text-sm font-medium">
+      {{ user.fullName }}
+    </div>
+    <div class="text-xs truncate w-40">
+      Good after noon! how can i help you?
```

Comment:
```suggestion
      Good afternoon! How can I help you?
```
i know it's just demo text but still :D